pkg/apis/config/v1alpha1/types.go

```diff
@@ -55,3 +58,13 @@ type ACME struct {
 	// +optional
 	PropagationTimeout *metav1.Duration `json:"propagationTimeout,omitempty"`
 }
+
+// Custom contains optional custom configuration for the cert-controller-manager deployment
+type Custom struct {
+	// PrecheckNameservers is the value for the `--issuer.precheck-nameservers` command line parameter
+	// +optional
+	PrecheckNameservers *string `json:"precheckNameservers,omitempty"`
+	// CACertificates are custom root certificates to be made available for the cert-controller-manager
+	// +optional
+	CACertificates *string `json:"caCertificates,omitempty"`
```

Comment:
Same here. Shall we validate the input? I know it's also not done for `PrivateKey` but probably good to fail early.

Reply:
I added some basic validation for correct PEM format